src/services/EdFi.DataManagementService.Api/Program.cs

```diff
@@ -3,21 +3,29 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using EdFi.DataManagementService.Api.Configuration;
 using EdFi.DataManagementService.Api.Infrastructure;
+using Microsoft.Extensions.Options;
+using System.Text.Json;
 
 var builder = WebApplication.CreateBuilder(args);
 builder.AddServices();
 
 var app = builder.Build();
 
 app.UsePathBase("/api");
+app.UseMiddleware<LoggingMiddleware>();
 app.UseRouting();
 
-app.MapGet("/", () => "Data Management Service");
+app.MapGet("/", () => Results.Ok("Data Management Service"));
 app.MapGet("/ping", () => Results.Ok(DateTime.Now));
 
-var logAppSettingsService = app.Services.GetRequiredService<LogAppSettingsService>();
-logAppSettingsService.LogToConsole();
+app.MapGet("/LogAppSettings", (IOptions<AppSettings> options, ILogger<Program> logger) =>
+{
+    logger.LogDebug($"Log while debugging");
+    logger.LogInformation(message: JsonSerializer.Serialize(options.Value));
+    return Results.Ok("Successfully logged Application Settings");
+});
```

Comment:
Is there a plan to take this out again in the future, before this code goes to production?

Reply:
Yes @stephenfuqua. I am keeping that endpoint just to validate the  IOptions and  ILogger !